### PR TITLE
mpc: Add version 0.22

### PR DESCRIPTION
**Pull request by bharatvaj**
- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/mpc.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "homepage": "Command-line music player client for mpd",
```
Comment (issaclin32) [inline bucket/mpc.json:2]: what?? shouldn't this be `description`

File: bucket/mpc.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "url": "https://www.musicpd.org/download/mpc/0/mpc-0.22-win32.zip",
+    "hash": "11ef51e8b40723637704aa01af8ab17664f63662f409f0d56f2a59a960688724",
+    "extract_dir": "mpc-0.22",
+    "bin": "mpc.exe",
+    "checkver": {
+        "url": "https://musicpd.org/download/mpc/0/",
+        "regex": "mpc-([\\d.]+)-win32.zip"
+    },
+    "autoupdate": {
+        "url": "https://musicpd.org/download/mpc/0/mpc-$version-win32.zip"
```
Comment (issaclin32) [inline bucket/mpc.json:18]:
```suggestion
        "url": "https://musicpd.org/download/mpc/0/mpc-$version-win32.zip",
        "extract_dir": "mpc-$version"
```

File: bucket/mpc.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    "suggest": {
+        "mpd": "mpd"
+    },
+    "url": "https://www.musicpd.org/download/mpc/0/mpc-0.22-win32.zip",
+    "hash": "11ef51e8b40723637704aa01af8ab17664f63662f409f0d56f2a59a960688724",
+    "extract_dir": "mpc-0.22",
+    "bin": "mpc.exe",
+    "checkver": {
+        "url": "https://musicpd.org/download/mpc/0/",
+        "regex": "mpc-([\\d.]+)-win32.zip"
```
Comment (issaclin32) [inline bucket/mpc.json:15]:
```suggestion
        "regex": "mpc-([\\d.]+)-win32.zip",
        "reverse": true
```

File: bucket/mpc.json (hunk 2 above)
Comment (issaclin32) [inline bucket/mpc.json:18]:
```suggestion
        "url": "https://musicpd.org/download/mpc/0/mpc-$version-win32.zip",
        "extract_dir": "mpc-$version"
```

File: bucket/mpc.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": "0.22",
+    "description": "Command-line music player client for mpd",
+    "homepage": "https://www.musicpd.org/clients/mpc",
```
Comment (issaclin32) [inline bucket/mpc.json:4]:
```suggestion
    "homepage": "https://www.musicpd.org/clients/mpc",
```
There is a trailing whitespace, which causes CI to fail.

**Review by issaclin32 — APPROVED** [review]

**Review by rashil2000 — APPROVED** [review]

**Pull request merged**